.stylelintrc.json

```diff
@@ -1,3 +1,10 @@
 {
-  "extends": ["stylelint-config-standard"]
+  "extends": ["stylelint-config-standard"],
```

Comment:
Per the notes in the commit, regarding this rule the font family in question that was throwing a font-family-no-[missing-generic-family-keyword](https://stylelint.io/user-guide/rules/font-family-no-missing-generic-family-keyword/) warning is an icon font and the glyph in question is specific only to that font pack so providing a generic substitution is rather pointless.  So I updated the lint rule based on the ignore option provided here:

https://github.com/stylelint/stylelint/pull/4656/files